packages/jest-haste-map/src/index.js

```diff
@@ -230,6 +231,18 @@ class HasteMap {
     });
   }
 
+  matchGlobs(globs: Array<Glob>, rootDir: Path): Promise<Array<Path>> {
```

Comment:
Can we call this something else? The other method is called `matchFiles` so this one should probably be called `matchFilesWithGlob` or something.